### PR TITLE
Moves the ci to gh-actions

### PR DESCRIPTION
**Pull request by taiar**

### CODE REVIEW

**Comment by smorimoto** [comment]
`.github/workflow.yml` -> `.github/workflows/workflow.yml`

**Comment by taiar** [comment]
> `.github/workflow.yml` -> `.github/workflows/workflow.yml`

Thanks :upside_down_face:

File: .github/workflows/workflow.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup asdf
+        uses: asdf-vm/actions/setup@v1
+
+      - name: Install dependencies
+        run: sudo apt-get -qq update; sudo apt-get -y install build-essential libcurl3-dev libreadline-dev gfortran liblzma-dev liblzma5 libbz2-1.0 libbz2-dev xorg-dev libbz2-dev liblzma-dev libpcre2-dev
```
Comment (smorimoto) [inline .github/workflows/workflow.yml:35]: Why not just move this to plugin_test job and remove the test job?

**Comment by taiar** [comment]
Thanks for your directions and sorry for the extra work here @smorimoto, first time setting up a gh action.

**Comment by smorimoto** [comment]
No need to apologize! Thank you for working on this!

**Pull request merged**